modules/pel/peltool/peltool.py

```diff
@@ -245,6 +245,78 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parsePELSummary(stream: DataStream, config: Config):
+    """
+    Parses and summarizes data from a PEL stream.
+    Returns: Event ID (eid) and summary of PEL data.
+            If no PEL is parsed or an error occurs, empty strings are returned.
+    """
+    out = OrderedDict()
+    ret, ph = generatePH(stream, out)
+    if ret is False:
+        return "", ""
+    eid = ph.pLID
+    ret, uh = generateUH(stream, ph.creatorID, out)
+    if ret is False:
+        return "", ""
+    if not uh.isServiceable():
```

Comment:
the C++ peltool also skips if the hidden action flag is set.  Usually hidden is always set on recovered or info PELs, so I think this is probably good enough, just something we need to keep an eye on.